src/components/buttons/LinkButton.js

```diff
@@ -9,16 +9,14 @@ import {
   buttonVariantProps,
   buttonDefaultProps,
 } from './Button'
-// Richard are we using this trackUserBehaviourProp on Google Analytics or Google Tag Manager?
-// import trackUserBehaviour, { CLICK_ON_CTA } from '../utils/trackUserBehaviour'
 import { ExternalLinkIcon, PdfDownload } from '../../components/icons'
 
 const fontColor = css`
   ${props => {
     const color = props.color
 
     return `
-    text-shadow: 0px 0px 1px ${color} !important;
+    color: ${color};
```

Comment:
Is there no more text shadow on link button? Or it’s coming from styles system now?

Reply:
@ric9176 just to double check it refers to text-shadow, not to box-shadow, so buttons have still shadow. This one made the bold text inside the buttons harder to read. I think this is something Horacio had pointed out a few weeks ago about some text being blurred. I've also noticed it today so I fixed it